src/common/GPOWrapper/GPOWrapper.cpp

```diff
@@ -176,4 +176,8 @@ namespace winrt::PowerToys::GPOWrapper::implementation
     {
         return static_cast<GpoRuleConfigured>(powertoys_gpo::getAllowedAdvancedPasteOnlineAIModelsValue());
     }
+    GpoRuleConfigured GPOWrapper::GetConfiguredNewEnabledValue()
```

Comment:
The same as on the policy name. This is confusing. Please rename to `NewPlus`.

Reply:
Thank you! I've finally gotten around to renaming to NewPlus throughout. Thanks again.